docs-v2/integrations/all/tapclicks.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Tapclicks
+sidebarTitle: Tapclicks
+---
+
+API configurations: [`tapclicks`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Request for API credentials](https://docs.google.com/forms/d/e/1FAIpQLSf7vy0PInJhU3CYPCQW6YFGCc8HgSfuOywLcih05zyfPVIFLg/viewform?vc=0&c=0&w=1&flr=0&gxids=7628)
```

Comment:
is there a doc that contains this link? I don't think we should make people trust a random google doc

Reply:
I agree, this can change quickly. I have updated it to include a link to the provider's documentation section, which contains the link to the Google Form.